src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
+html {
+  font-family: "Roboto", sans-serif;
+  --link-color: #00acdc;
+  --text-color: #060b35;
+}
+
 body {
   margin: 0;
 }
+
+.container {
+  display: block;
+  width: 198px;
+  height: 406px;
```

Comment:
```suggestion

```
It's a bad practice to hardcode the height of the element, it should get the height from its content